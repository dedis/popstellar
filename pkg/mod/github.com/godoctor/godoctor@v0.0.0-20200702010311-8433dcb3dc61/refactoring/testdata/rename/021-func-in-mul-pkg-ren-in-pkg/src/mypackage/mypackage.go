package mypackage 
func MyFunction(n int) int {    // <<<<< rename,2,6,2,6,Xyz,pass         
	if n == 0 {
		return 1
	} else {
		return n + MyFunction(n-1)
	}
}
