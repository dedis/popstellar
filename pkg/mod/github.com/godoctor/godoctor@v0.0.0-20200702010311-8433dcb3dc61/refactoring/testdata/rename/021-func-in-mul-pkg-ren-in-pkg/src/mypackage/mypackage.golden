package mypackage 
func Xyz(n int) int {    // <<<<< rename,2,6,2,6,Xyz,pass         
	if n == 0 {
		return 1
	} else {
		return n + Xyz(n-1)
	}
}
